README.md

```diff
@@ -14,6 +14,75 @@ Thumbprint Tokens are published as JavaScript, SCSS, Swift, and Kotlin.
 
 You can use the [Thumbprint Tokens GraphQL server](https://thumbprint-tokens.netlify.com/) to query tokens and view the schema.
 
+Button, Theme, Title Color, Active, primary Tokens should follow this recommended hierarchy: `[<component-name>] <category-name> [<option-name> <state>] <token>`.
```

Comment:
How would it look for tokens that are not related to options but we still want to keep consistent on all platforms? Imagine the CornerRadius of a button:

```
Button-CornerRadius
```